.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Thanks for showing me how to use eslint, it seemed confusing until you broke it down for me in a stand-up.